tests/api-mongo.test.js

```diff
@@ -38,6 +40,8 @@ import { expectUniqSetsOfResponses, expectArrayIsInOrder } from './shared/utils'
 const { expect } = chai;
 const { ObjectId } = mongoose.Types;
 
+chai.use(chaiHttp);
```

Comment:
You don't need to add this line since you made a new command in`commands.js` that imports `chai` and `chaiHttp`